package.json

```
@@ -29,7 +29,7 @@
     "@formatjs/intl-pluralrules": "^5.2.4",
     "@formatjs/intl-relativetimeformat": "^11.2.4",
     "@gorhom/bottom-sheet": "^4.5.1",
-    "@mapeo/ipc": "^0.2.0",
+    "@mapeo/ipc": "^0.3.0",
```

maybe pin this to exact version?

```suggestion
    "@mapeo/ipc": "0.3.0",
```